autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -117,6 +117,20 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             Use `auto_stack` to maximize predictive accuracy; use `hyperparameter_tune` if you prefer to deploy just a single model rather than an ensemble.
         feature_prune : bool, default = False
             Whether or not to perform feature selection.
+        refit_full : bool, default = False
+            Whether to retrain all models on all of the data after the normal training procedure.
```

Comment:
maybe clarify: "on all of the data (training+validation) after ..."

Reply:
Added